package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
I can try to use shell vars to remember the first `onSuccess`, to ignore any future triggering. But not sure it is a good idea. Also probably hard to make it work crossplatform.